src/main/java/Command/Command.java

```diff
@@ -0,0 +1,41 @@
+package Command;
+import Duke.DukeUi;
+import Duke.DukeException;
+import Duke.Storage;
+import Duke.TaskList;
+
+/**
+* Command is an abstract class that serves as a blueprint for all user commands parsed by Parser Class.
+*
+* @author Linus Chui
+*/
+public abstract class Command {
+
+    /**
+     * The exit boolean decides if the command exits the program.
+     */
```

Comment:
May I ask if you intend to write it as comment/comment block or as a JavaDocs? If it is a comment, it should be 
```
// comment 
or
/* 
text
*/
```